### PR TITLE
Add docs for virtual machine probes

### PR DESCRIPTION
**Pull request by howardjohn**
This is built on top of https://github.com/istio/istio.io/pull/8746/files since it is easier to wait for that one

### CODE REVIEW

File: content/en/docs/setup/install/virtual-machine/index.md [hunk 1]
```diff
@@ -249,6 +249,35 @@ Install Istio and expose the control plane so that your virtual machine can acce
     $ kubectl --namespace ${VM_NAMESPACE} apply -f workloadgroup.yaml
     {{< /text >}}
 
+    When using the Automated WorkloadEntry Creation feature, application health checks are also available. These share the same API and behavior as [Kubernetes Readiness Probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/).
+
+    For example, to configure a probe on the `/ready` endpoint of your application:
+
+    {{< text bash >}}
```
Comment (howardjohn) [inline content/en/docs/setup/install/virtual-machine/index.md:256]: This, and the above, end up broken:
```
$ cat <<EOF > workloadgroup.yaml
apiVersion: networking.istio.io/v1alpha3
kind: WorkloadGroup
metadata:
  name: "${VM_APP}"
  namespace: "${VM_NAMESPACE}"
spec:
  metadata:
labels:
  app: "${VM_APP}"
  template:
serviceAccount: "${SERVICE_ACCOUNT}"
network: "${NETWORK}"
  probe:
periodSeconds: 5
initialDelaySeconds: 1
httpGet:
  port: 8080
  path: /ready
EOF
```

the spacing is all wrong. Any ideas?

Comment (ericvn) [reply content/en/docs/setup/install/virtual-machine/index.md:256]: I suspect it has to do with spaces vs tabs, but would need to look closer.

Comment (ericvn) [reply content/en/docs/setup/install/virtual-machine/index.md:256]: Just looking at the particular file, my editor has the indention figured to be 4 spaces, while other files have it being 2 spaces. So something in the files relative to indention. I know @frankbu has chased issues with indentation and things looking incorrect and he probably knows what to fix.

Comment (frankbu) [reply content/en/docs/setup/install/virtual-machine/index.md:256]: Hmm, it's a complicated nesting of things: text code block inside a tabset inside a numbered list. The second tab even has another nested number list in it. Maybe it's a generator bug? I would try experimenting with simpler structures, worst case, take out the numbered list and use subsections for the steps or drop the tabset.

Comment (stevenctl) [reply content/en/docs/setup/install/virtual-machine/index.md:256]: I ended up removing the numbered list, seems to work: https://deploy-preview-8755--preliminary-istio.netlify.app/latest/docs/setup/install/virtual-machine/

Comment (howardjohn) [reply content/en/docs/setup/install/virtual-machine/index.md]: ok I think i have it fixed, thanks for the help

**Review by stevenctl — APPROVED** [review]
content LGTM

**Review by ericvn — APPROVED** [review]

File: content/en/docs/setup/install/virtual-machine/index.md [hunk 2]
```diff
@@ -250,6 +250,35 @@ Then, to allow automated `WorkloadEntry` creation, push the `WorkloadGroup` to t
...
+metadata:
+  name: "${VM_APP}"
+  namespace: "${VM_NAMESPACE}"
+spec:
+  metadata:
+    labels:
+      app: "${VM_APP}"
+  template:
+    serviceAccount: "${SERVICE_ACCOUNT}"
+    network: "${NETWORK}"
```
Comment (frankbu) [inline content/en/docs/setup/install/virtual-machine/index.md:270]: Shouldn't this be `${VM_NETWORK}`?

**Pull request merged**